docs/open-source-project/release-notes/tizen-5-0-m2.md

```diff
@@ -0,0 +1,468 @@
+  - Text
+    - Support for software styling has been added.
+  - Timer
+    - APIs to pause and resume the timer have been added.
+  - Control and Visual
+    - WebView control has been added.
+    - Support for FlexLayout, AbsoluteLayout and GridLayout has been added.
+  - 3D Model
+    - Support for glTF (GL Transmission Format) has been added.
+  - Performance
+    - Changed not to render transparent texts.
+  - NUI (C# Interface)
+    - Added support for WebView.
+- SDL (Simple DirectMedia Layer)
+  - Upgrade to 2.0.8.
+  - Added SDL sub-module as SDL-ttf.
+  - Added new features.
+    - Auxiliary Hint.
+    - Focus Skip.
+
+### Fixes
+
+- DALi (3D UI Toolkit)
+  - Actor and Renderer
+    - RenderTask bug to waiting forever has been fixed.
+    - Depth buffer clear bug has been fixed.
+    - BlendFunc has been fixed in case of a non-premultiplied format.
+  - Window
+    - Flickering issue has been fixed when a window is resized.
+  - Control
+    - Some bugs about layout have been fixed.
+  - NUI (C# Interface)
+    - Minor bugs (ex: wrong position, not shown) of Layout have been fixed.
+    - RaiseToTop in View class bug has been fixed.
+    - XamlResource memory leak error has been fixed.
+    - VisualFittingMode bug in VisualMaps has been fixed.
+- Evas Rendering Engine.
+  - Bug of rotation is fixed in Evas GL Engine.
+  - Added thread safety patch to Evas TBM Engine.
+  - Bug of image downscaling is fixed in Evas GL Engine.
+  - Added the patch is TBM Queue is used per TBM surface in Evas TBM Engine.
+-  SDL (Simple DirectMedia Layer)
+  - Bug of pause/resume in multi-windows is fixed.
+
+
+## UI framework
+
+### New and changed features
+
+- EFL
+  - Version 1.20 is upgraded to 1.21.
+- Harfbuzz
+  - Version 0.4.6 is upgraded to 1.8.1.
+- Freetype2
+  - Version 2.9 is upgraded to 2.9.1.
+- Text Input
+  - NLP (Natural Language Processing) API has been added.
+- Voice interaction
+  - Added new APIs to supporting multiple assistant.
+  - Added APIs which has platform privilege to supporting voice panel app. 
+  - Added APIs to supporting to send and to receive specific engine result.
+
+### Fixes
+
+- Fixed bugs.
+- CBHM (Clip Board History Manager) is now working properly.
+
+
+## Multimedia framework
+
+### New and changed features
+
+- Common
+  - Tizen Allocator & Bufferpool has been added.
+- Media Player
+  - Playback time related APIs in nanoseconds has been added.
+  - Setting zoom level with fov about spherical video API has been added.
+  - Video ROI (region of interest) API has been added.
+  - Support for resuming HTTP playback during buffering has been added.
+  - Support for cancellation of asynchronous preparing has been allowed.
+  - Support for some tags of HLS v20 has been added.
+  - The tizen allocator is applied to mediacodec.
+  - The tizen allocator & buffer pool have been applied to the following modules:
+    - Player, MediaCodec, Video360, VideoConverter

```

Comment:
Rephrase as:
Player, MediaCodec, Video360, and VideoConverter

Reply:
will accept